scripts/src/java/org/oppia/android/scripts/proto/coverage.proto

```diff
@@ -0,0 +1,63 @@
+syntax = "proto3";
+
+package proto;
+
+option java_package = "org.oppia.android.scripts.proto";
+option java_multiple_files = true;
+
+message CoverageReport {
+  string bazel_test_target = 1;
+  repeated CoveredFile covered_file = 2;
+}
+
+message CoveredFile {
+  string file_path = 1;
+  string file_sha1_hash = 2;
+  repeated CoveredLine covered_line = 3;
+  optional int32 lines_found = 4;
+  optional int32 lines_hit = 5;
+  repeated FunctionCoverage function_coverage = 6;
+  optional int32 functions_found = 7;
+  optional int32 functions_hit = 8;
+  repeated BranchCoverage branch_coverage = 9;
+  optional int32 branches_found = 10;
+  optional int32 branches_hit = 11;
+}
+
+message CoveredLine {
+  int32 line_number = 1;
+  CoveredLine.Coverage coverage = 2;
+
+  enum Coverage {
+    UNSPECIFIED = 0;
+    FULL = 1;
+    NONE = 2;
+  }
+}
+
+message BranchCoverage {
+  int32 line_number = 1;
+  optional int32 block_number = 2;
```

Comment:
Drop all optionals in this file--optional is the default (and only) property for proto3.

Reply:
So @BenHenning, Can I just understand if this would affect storing and accessing values in our proto? Initially, I used int32 for execution_count, hit_count, block_number, branch_number, etc., but since 0 is considered a default value, these fields are not actually represented in the proto. This resulted in:

```
functions_found: 11
functions_hit: 7
branch_coverage {
  line_number: 46
  block_number: 1
  hit_count: 1
}
branch_coverage {
  line_number: 46
  branch_number: 1 // hit_count not shown as 0 (default)
}
```
Would this cause issues when setting Coverage as None or when trying to parse the data into HTML? Sorry if I missed any alternative ways to retrieve the data while still ignoring optional fields. I'll look into the available docs further in meantime, but would having no value representation be considered as 0 and parsed accordingly?

Reply:
Removed the optional specifications from all int32 as they just work as intended.